### PR TITLE
Adding the migration to include the project.

### PR DESCRIPTION
**Pull request by maxyuri13**
- Added migrations for adding the project.
- Established the connection between adding the project and the user.

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -15,4 +15,19 @@ model User {
   password_hash String
   created_at    DateTime @default(now())
   updated_at    DateTime @updatedAt
+
+  addProjects   AddProject[]
 }
+
+model AddProject {
```
Comment (MatheusSanchez) [inline prisma/schema.prisma:22]: Wouldn't it be projects?

File: prisma/schema.prisma [hunk 2]
```diff
@@ -15,4 +15,19 @@ model User {
   password_hash String
   created_at    DateTime @default(now())
   updated_at    DateTime @updatedAt
+
+  addProjects   AddProject[]
```
Comment (MatheusSanchez) [inline prisma/schema.prisma:19]: Wouldn't it be projects?

**Review by MatheusSanchez — CHANGES_REQUESTED** [review]

File: prisma/migrations/20240126134916_orange_db_portfolio/migration.sql [hunk 3]
```diff
@@ -0,0 +1,32 @@
+-- CreateTable
+CREATE TABLE "User" (
+    "id" TEXT NOT NULL,
+    "name" TEXT NOT NULL,
+    "surname" TEXT NOT NULL,
+    "email" TEXT NOT NULL,
```
Comment (MatheusSanchez) [inline prisma/migrations/20240126134916_orange_db_portfolio/migration.sql:6]: Why is that here? That is already done in the first migration that you delete!

**Review by MatheusSanchez — CHANGES_REQUESTED** [review]

File: prisma/schema.prisma (hunk 1 above)
Comment (maxyuri13) [reply prisma/schema.prisma]: Done

File: prisma/schema.prisma (hunk 2 above)
Comment (maxyuri13) [reply prisma/schema.prisma]: Done.

**Review by MatheusSanchez — APPROVED** [review]
